src/views/SimulationAutocalsolView.vue

```diff
@@ -54,12 +58,23 @@ onMounted(async () => {
     peak_power: getPeakPower(),
   }
 
-  state.autocalsolResult = await apiAutocalsolService.getComputeData(
-    state.dataAutocalsol
-  )
-  autocalsolStore.setAutocalsolResult(state.autocalsolResult)
-  console.log(state.autocalsolResult)
-  router.push('/simulation-results')
+  try {
+    state.autocalsolResult = await apiAutocalsolService.getComputeData(
+      state.dataAutocalsol
+    )
+    autocalsolStore.setAutocalsolResult(state.autocalsolResult)
+    if (autocalsolStore.autocalsolResult === null) {
```

Comment:
Here can you add a check to compare the type of autocalsolStore.autocalsolResult and AutocalsolResult please